### ISSUE TITLE
Add deployment scripts for common server setups

### ISSUE THREAD

**Issue by 9b** [blockadeio/cloud_node#10]
The default flask server is fine for development, but should not be used inside of production. Ideally, we would have at least one build script that automatically sets up the cloud node to run on a real webserver. Given the popularity of Apache, that would be a good place to start.

**Comment by villain** [comment]
nginx might be the better option. using something like apparmor on ubuntu probably isn't a bad idea in this scenario either

**Comment by blockadeio** [comment]
I agree with you. Nginx offers a much smaller footprint with some good ways to scale back to Python applications. Any interest in taking this on @villain?
